tools/github/client.go

```diff
@@ -110,9 +111,9 @@ func CacheFilePath() (string, error) {
 }
 
 var (
-	// Correctly formatted FerretDB issue.
-	// It returns repository name and the issue number as it's submatches.
-	urlRE = regexp.MustCompile(`^\Qhttps://github.com/FerretDB/\E([-\w]+)/issues/(\d+)$`)
+	// Checks correctly formatted FerretDB issue and microsoft/documentdb issue.
+	// It returns owner/repository name and the issue number as it's submatches.
+	urlRE = regexp.MustCompile(`^\Qhttps://github.com/\E(FerretDB/[-\w]+|microsoft/documentdb)/issues/(\d+)$`)
```

Comment:
Hm, I think it is strange that we have that regex in 3 places. Like, we check if something matches regexp, then pass it there, where we check regex again? Given that regexes are the same now, we might simplify that

Reply:
Thanks that's true, it makes sense to pass them so I removed regex check in client.go.